src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadUuidFromPostgreSQL.java

```diff
@@ -0,0 +1,71 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+import com.northconcepts.datapipeline.core.DataWriter;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.internal.test.DataPipelineUnitTest;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.job.Job;
+
+public class ReadUuidFromPostgreSQL extends DataPipelineUnitTest {
+
+    final static String DATABASE_DRIVER = "org.postgresql.Driver";
+    final static String DATABASE_URL = "jdbc:postgresql://localhost:5432/contacts?&stringtype=unspecified";
```

Comment:
Did you intend to put both "?" and "&"?  Please double check the syntax since this isn't a web URL.

Reply:
My bad during copy-paste. 🤦  Fixed the syntax and validated with the test container.